JH: Add sections for logging into Guacamole VDI and connecting to VM …

jhay-epcc opened the pull request:
…in docs/access/virtualmachines-vdi.md

# EIDF Documentation Pull Request

## Description

Please include a summary of the change and any issues being addressed.

Please also include relevant motivation and context.

List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New Documentation

## What has to be reviewed

List the pages/sections that are to be reviewed.

## Checklist

- [x] Documentation follows the project style guidelines
- [x] Ensure Contact details contain Service Emails and Numbers
- [x] Self-review of documentation using mkdocs on local system
- [x] Spellcheck has been performed
- [] Pre-commit has been run and passed

agngrant commented:
Can the information on getting a SAFE account be moved to a specific SAFE account page? rather than at the top of the VDI access page - possible in the Access Overview page or as "Getting a SAFE Account" 

The long redirect URL I think is going to confused people - possibly just say that it will redirect you to SAFE? rather than give the URL with <NONCE_ID>.

A nice if would be to markup the images to have indications of what each thing is - such as this is a list of VMs?

akrause2014 commented:
Agree that the information on registering with SAFE should be moved to another page. It should also link to the SAFE documentation https://epcced.github.io/safe-docs/ . In particular: Register https://epcced.github.io/safe-docs/safe-for-users/#register and Join a Project https://epcced.github.io/safe-docs/safe-for-users/#how-to-request-to-join-a-project - maybe with some custom instructions. 
The training course instructions should be in a completely separate tree because they don't apply to EIDF users in general. When EIDF users register for SAFE they should be using their own institution and department and not EIDF and EPCC.
Everything below "Login to the EIDF VDI" can stay where it is I would say, everything above should be moved to the Workshops section, maybe with a page for each workshop, depending whether it's JupyterHub or VDI/Openstack VM based.

jhay-epcc commented:
1. Information on getting a SAFE account be moved to a specific SAFE account page: access/workshops/safe-registration.md
2. Removed the long redirect URL
3. Added figure captions
4. Everything below "Login to the EIDF VDI" stays in access/virtualmachines-vdi.md
5. Everything above "Login to the EIDF VDI" moved into access/workshops/safe-registration.md
6. Separate pages for workshops: access/workshops/workshops.md and access/workshops/jhub-git.md for Notebook-based. Will add another page for VM-based workshops in a subsequent PR.

akrause2014 commented:
In the future, consider moving the SAFE registration page to a central location outside the workshop pages as there's an overlap with the portal documentation.
